aip/general/0193.md

```diff
@@ -59,13 +47,17 @@ following:
       {
         "@type": "type.googleapis.com/google.rpc.LocalizedMessage",
         "locale": "en-US",
-        "message": "An <e2-medium> VM instance with <local-ssd=3,nvidia-t4=2> is currently unavailable in the <us-east1-a> zone.\n Consider trying your request in the <us-central1-f,us-central1-c> zone(s), which currently has/have capacity to accommodate your request.\n Alternatively, you can try your request again with a different VM hardware configuration or at a later time. For more information, see the troubleshooting documentation."
+        "message": "An <e2-medium> VM instance with <local-ssd=3,nvidia-t4=2> is currently unavailable in the <us-east1-a> zone.
```

Comment:
this is invalid json, the value inside quotes must be declared on a single line.